man/dracut.8.asc

```diff
@@ -391,8 +391,8 @@ Default:
 
 **--persistent-policy** _<policy>_::
     Use _<policy>_ to address disks and partitions.
-    _<policy>_ can be any directory name found in /dev/disk.
-    E.g. "by-uuid", "by-label"
+    _<policy>_ can be any directory name found in /dev/disk (e.g. "by-uuid",
+    "by-label"), or "mapper" to use /dev/mapper device names (default).
```

Comment:
This man page is for all dracut deployment. Is "mapper" the default for all dracut installations ?